core/app/models/spree/customer_return.rb

```diff
@@ -1,5 +1,6 @@
 module Spree
   class CustomerReturn < Spree::Base
+    include Spree::Core::NumberGenerator.new(prefix: 'CR', length: 9)
```

Comment:
There should be no need to overwrite. AFAIK there is no case where you subclass a model that already includes this "instantiated mixin", where you want to give the subclass a different number generator.

@JDutil Does this answer your question?